startup.py

```diff
@@ -102,19 +102,19 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
             # Append executable folder to PATH environment variable
             server_python_exe = plugin_env.get("ALIAS_PLUGIN_SERVER_PYTHON")
             if server_python_exe:
-                sgtk.util.append_path_to_env_var(
+                sgtk.util.prepend_path_to_env_var(
                     "PATH", os.path.dirname(server_python_exe)
                 )
             else:
-                sgtk.util.append_path_to_env_var(
+                sgtk.util.prepend_path_to_env_var(
```

Comment:
I guess this fix the wrong picking of executable Alias was doing.

Reply:
Yes, that is correct